src/components/tagline.jsx

```diff
@@ -5,20 +5,24 @@ import PropTypes from 'prop-types';
 
 function Tagline({ tagline }) {
   return (
-    <GridContainer className="usa-section">
-      <Grid row gap>
-        {tagline.heading && (
-          <Grid tablet={{ col: 4 }}>
-            <h2 className="font-heading-xl">{tagline.heading}</h2>
-          </Grid>
-        )}
-        {tagline.text && (
-          <Grid className="usa-prose" tablet={{ col: 8 }}>
-            <p>{tagline.text}</p>
-          </Grid>
-        )}
-      </Grid>
-    </GridContainer>
+    <section className="usa-section">
+      <GridContainer className="usa-section">
+        <Grid row gap>
+          {tagline.heading && (
+            <Grid className="tagline__heading-container" tablet={{ col: 4 }}>
+              <h2 className="font-heading-xl tagline__heading">
```

Comment:
Do lines 12 and 13 follow BEM? Maybe the class in line 12 should be `tagline__heading" and the class in line 13 should be `tagline__heading-text`?

Reply:
Good suggestion!